### PR TITLE
Added function to render to pipeable stream

### PR DESCRIPTION
**Pull request by wredzio**

### CODE REVIEW

File: Forte.React.AspNetCore/Forte.React.AspNetCore.csproj [hunk 1]
```diff
@@ -21,14 +21,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jering.Javascript.NodeJS" Version="6.3.1" />
+    <PackageReference Include="Jering.Javascript.NodeJS" Version="7.0.0-beta.4" />
```
Comment (pawil31) [inline Forte.React.AspNetCore/Forte.React.AspNetCore.csproj:24]: why do we use beta?

File: Forte.React.AspNetCore/React/ReactService.cs [hunk 2]
```diff
@@ -69,13 +77,55 @@ Stream ModuleFactory()
...
+            return;
+        }
+
+        var result = await InvokeRenderTo<HttpResponseMessage>(component, props,
+            writeOutputHtmlToOptions ?? new WriteOutputHtmlToOptions());
+
+        using var reader = new StreamReader(await result.Content.ReadAsStreamAsync());
+        int character;
+
+        while ((character = reader.Read()) != -1)
```
Comment (pawil31) [inline Forte.React.AspNetCore/React/ReactService.cs]: ReadAsync(...

Comment (pawil31) [inline Forte.React.AspNetCore/React/ReactService.cs]: consider using some buffer.

File: Forte.React.AspNetCore/Forte.React.AspNetCore.csproj (hunk 1 above)
Comment (pawil31) [reply Forte.React.AspNetCore/Forte.React.AspNetCore.csproj:24]: Consider releasing our package as beta

Comment (konrad-galczynski) [inline Forte.React.AspNetCore/Forte.React.AspNetCore.csproj:24]: are we fine with using beta packages?

Comment (wredzio) [reply Forte.React.AspNetCore/Forte.React.AspNetCore.csproj:24]: I will release my package as beta as well

**Review by konrad-galczynski — APPROVED** [review]

File: Forte.React.AspNetCore/React/ReactService.cs [hunk 3]
```diff
@@ -69,13 +77,57 @@ Stream ModuleFactory()
...
+        {
+            return;
+        }
+
+        var result = await InvokeRenderTo<HttpResponseMessage>(component, props,
+            writeOutputHtmlToOptions ?? new WriteOutputHtmlToOptions());
+
+        using var reader = new StreamReader(await result.Content.ReadAsStreamAsync());
+
+        char[] buffer = new char[1024];
```
Comment (pawil31) [inline Forte.React.AspNetCore/React/ReactService.cs:120]: please use stackallock

**Review by pawil31 — APPROVED** [review]

File: Forte.React.AspNetCore/React/ReactService.cs (hunk 3 above)
Comment (wredzio) [reply Forte.React.AspNetCore/React/ReactService.cs:120]: image

**Pull request merged**